src/main/java/org/apache/commons/codec/binary/Base64.java

```diff
@@ -127,6 +127,58 @@ public Builder setUrlSafe(final boolean urlSafe) {
             return setEncodeTable(toUrlSafeEncodeTable(urlSafe));
         }
 
+        /**
+         * Sets the format of the decoding table.
```

Comment:
This method and `Builder.setUrlSafe(boolean)` should make it clear that it doesn'r affect the action of the other. For example, `Builder.setUrlSafe(boolean)` already says that it only affects encoding but we should also say that if you care about _decoding_ then you should call... I tihnk we want to make it clear what to call for what use case. It might be worth mentioning at the class level and Builder class level as well.